<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="cuenta_ctacte_histAvencer-20240618123858" tests="2" file=".py" time="96.036" timestamp="2024-06-18T12:40:34" failures="0" errors="2" skipped="0">
	<testcase classname="cuenta_ctacte_histAvencer" name="test_ctacte_historica" time="0.000" timestamp="0001-01-01T00:00:00" file="CtaCte_HistoAvenver.py" line="17">
		<error type="AttributeError" message="'NoneType' object has no attribute 'text'"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\CtaCte_HistoAvenver.py", line 115, in test_ctacte_historica
    validate_character_numeric_element(self.driver, balance_ars)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 1147, in validate_character_numeric_element
    valor = elemento.text
AttributeError: 'NoneType' object has no attribute 'text'
]]></error>
		<system-out><![CDATA[¡Elemento encontrado y clickeado 6 veces con éxito!
¡Elemento encontrado y clickeado 3 veces con éxito!
Tiempo de espera agotado. El texto por xpaht no está presente.
]]></system-out>
	</testcase>
	<testcase classname="cuenta_ctacte_histAvencer" name="test_ctacte_historica" time="96.036" timestamp="2024-06-18T12:40:34" file="CtaCte_HistoAvenver.py" line="17">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=125.0.6422.176)
Stacktrace:
	GetHandleVerifier [0x00007FF664C41F52+60322]
	(No symbol) [0x00007FF664BBCEC9]
	(No symbol) [0x00007FF664A77EBA]
	(No symbol) [0x00007FF664A4D5A5]
	(No symbol) [0x00007FF664AF36B7]
	(No symbol) [0x00007FF664AFA812]
	(No symbol) [0x00007FF664AEC1E0]
	(No symbol) [0x00007FF664AB9617]
	(No symbol) [0x00007FF664ABA211]
	GetHandleVerifier [0x00007FF664F594AD+3301629]
	GetHandleVerifier [0x00007FF664FA36D3+3605283]
	GetHandleVerifier [0x00007FF664F99450+3563680]
	GetHandleVerifier [0x00007FF664CF4326+790390]
	(No symbol) [0x00007FF664BC750F]
	(No symbol) [0x00007FF664BC3404]
	(No symbol) [0x00007FF664BC3592]
	(No symbol) [0x00007FF664BB2F9F]
	BaseThreadInitThunk [0x00007FFBED37257D+29]
	RtlUserThreadStart [0x00007FFBEECEAA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\CtaCte_HistoAvenver.py", line 196, in tearDown
    self.driver.close()
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 459, in close
    self.execute(Command.CLOSE)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=125.0.6422.176)
Stacktrace:
	GetHandleVerifier [0x00007FF664C41F52+60322]
	(No symbol) [0x00007FF664BBCEC9]
	(No symbol) [0x00007FF664A77EBA]
	(No symbol) [0x00007FF664A4D5A5]
	(No symbol) [0x00007FF664AF36B7]
	(No symbol) [0x00007FF664AFA812]
	(No symbol) [0x00007FF664AEC1E0]
	(No symbol) [0x00007FF664AB9617]
	(No symbol) [0x00007FF664ABA211]
	GetHandleVerifier [0x00007FF664F594AD+3301629]
	GetHandleVerifier [0x00007FF664FA36D3+3605283]
	GetHandleVerifier [0x00007FF664F99450+3563680]
	GetHandleVerifier [0x00007FF664CF4326+790390]
	(No symbol) [0x00007FF664BC750F]
	(No symbol) [0x00007FF664BC3404]
	(No symbol) [0x00007FF664BC3592]
	(No symbol) [0x00007FF664BB2F9F]
	BaseThreadInitThunk [0x00007FFBED37257D+29]
	RtlUserThreadStart [0x00007FFBEECEAA48+40]

]]></error>
		<system-out><![CDATA[¡Elemento encontrado y clickeado 6 veces con éxito!
¡Elemento encontrado y clickeado 3 veces con éxito!
Tiempo de espera agotado. El texto por xpaht no está presente.
]]></system-out>
	</testcase>
</testsuite>
